MBBSEmu/HostProcess/HostRoutines/MenuRoutines.cs

```diff
@@ -178,7 +182,14 @@ private void WelcomeScreenDisplay(SessionBase session)
         {
             EchoToClient(session, new byte[] { 0x1B, 0x5B, 0x32, 0x4A });
             EchoToClient(session, new byte[] { 0x1B, 0x5B, 0x48 });
-            EchoToClient(session, _resourceManager.GetResource("MBBSEmu.Assets.login.ans").ToArray());
+            string LoginFileName = _configuration["Login.File"];
```

Comment:
I think we should rename the key to `ANSI.Login` only because I think we'll have an opportunity to use it in a couple more places, like `ANSI.Singup` and perhaps `ANSI.Logoff` and it'd keep them alphabetically grouped.